### PR TITLE
Update install.md

### PR DESCRIPTION
**Pull request by Jorc0**
I had to configure the path to my flarum, and it didnt work untill i changed the config.php
I can be wrong

### CODE REVIEW

**Comment by SychO9** [comment]
Isn't the URL filled in the installation process through the installer 🤔 ?

Also I suggest: (edit starting from ``Finally, edit the `site.php` and updat..``
````md
 Edit the `site.php` and update the paths in the following lines to reflect your new directory structure:

```php
'base' => __DIR__,
'public' => __DIR__,
'storage' => __DIR__.'/storage',
```

Finally, check `config.php` and make sure the `url` value is correct.
````

**Comment by Jorc0** [comment]
> Isn't the URL filled in the installation process through the installer 🤔 ?
> 
> Also I suggest: (edit starting from `` Finally, edit the `site.php` and updat.. ``
> 
> ```lisp
>  Edit the `site.php` and update the paths in the following lines to reflect your new directory structure:
> 
> ```php
> 'base' => __DIR__,
> 'public' => __DIR__,
> 'storage' => __DIR__.'/storage',
> ```
> 
> Finally, check `config.php` and make sure the `url` value is correct.
> ```

Modified

File: docs/install.md [hunk 1]
```diff
@@ -121,14 +121,18 @@ You will also need to edit the `index.php` file and change the following line:
...
 ```php
 'base' => __DIR__,
 'public' => __DIR__,
 'storage' => __DIR__.'/storage',
 ```
+Once you done everything properly, you have to edit `config.php` line 18 to the following:
 
+```
+'url' => 'domain.com/"index.php directory"/',
+```
```
Comment (SychO9) [inline docs/install.md:135]:
````suggestion
 Edit the `site.php` and update the paths in the following lines to reflect your new directory structure:

```php
'base' => __DIR__,
'public' => __DIR__,
'storage' => __DIR__.'/storage',
```

Finally, check `config.php` and make sure the `url` value is correct.
````

File: docs/install.md [hunk 2]
```diff
@@ -121,14 +121,15 @@ You will also need to edit the `index.php` file and change the following line:
...
- Finally, edit the `site.php` and update the paths in the following lines to reflect your new directory structure:
+ Edit the `site.php` and update the paths in the following lines to reflect your new directory structure:
 
 ```php
 'base' => __DIR__,
 'public' => __DIR__,
 'storage' => __DIR__.'/storage',
 ```
 
+Finally, check `config.php` and make sure the `url` value is correct.
```
Comment (SychO9) [inline docs/install.md:132]:
```suggestion
Finally, check `config.php` and make sure the `url` value is correct.

```

**Review by SychO9 — APPROVED** [review]
Thank You!

**Pull request merged**